creditcards/templates/partials/_creditcards_list.html

```diff
@@ -0,0 +1,28 @@
+{% load bootstrap3 %}
+{% block content %}
+<table>
+    <tbody>
+        {% for creditcard in creditcards_list %}
+            <tr>
+                <td>
+                    <a href="{% url 'users:creditcardbound' user.id creditcard.id %}">{{ creditcard.name }}&nbsp&nbsp</a>
+                </td>
```

Comment:
`&nbsp&nbsp` missing semicolons. Should be ``&nbsp;&nbsp;`

Reply:
fixed